src/Sylius/Behat/Context/Setup/OrderContext.php

```diff
@@ -210,7 +210,7 @@ public function theCustomerChoseShippingWithPayment(
      */
     public function theCustomerBoughtSingleProduct(ProductInterface $product)
     {
-        $this->addProductVariantToOrder($product->getMasterVariant(), $product->getPrice(), 1);
+        $this->addProductVariantToOrder($product->getFirstVariant(), $product->getFirstVariant()->getPrice(), 1);
```

Comment:
Won't it be better to leave `getPrice` method on product? We would have SSoT, and if anybody would decide that he need another logic for price displaying rather then this from the first variant, one could override it easily.